nautobot/ui/src/components/Navbar.js

```diff
@@ -96,13 +160,18 @@ export function Navbar() {
                     </ReactRouterNavLink>
                 ))}
             </NavbarSections>
-
-            <InputGroup flex="1" size="lg" variant="navbar">
-                <InputLeftElement>
-                    <SearchIcon />
-                </InputLeftElement>
-                <Input placeholder="Search..." />
-            </InputGroup>
+            <Tooltip
+                label="Global search is not yet implemented."
+                placement="bottom"
+                isDisabled={app_label && model_name ? true : false} // Disable tooltip if on a list or detail view (contextual search is implemented)
```

Comment:
Bool cast is a more elegant way of resolving given condition to either `true` or `false` in JavaScript.

```suggestion
                isDisabled={Boolean(app_label && model_name)} // Disable tooltip if on a list or detail view (contextual search is implemented)
```